### ISSUE TITLE
Better support for VSC

### ISSUE THREAD

**Issue by poesel** [hiveeyes/terkin-datalogger#71]
Now that all code is in /src there is only one step missing to be able to seamlessly use VSC: either move dist-packages into src (there is probaly some thelogical reason not to do this ;) ) or just link the directory into src (that is what I do).

**Comment by amotl** [comment]
Dear @poesel,

we can well move `dist-packages` into the `src` folder and finally add a `pymakr.conf` (#32).

With kind regards,
Andreas.

**Comment by poesel** [comment]
If dist-packages is in /src there is no need for a pymakr.conf because everything in /src will have to be synced to the MCU. Well except if you want to exclude syncing the __pycache__ directories that are created through 'make test'. But since its only the directories and not their contents its a mere nuisance.

**Comment by poesel** [comment]
so are we going to move /dist-packages into /src or not?

**Comment by poesel** [comment]
?

**Comment by amotl** [comment]
Dear @poesel,

we might well consider that in the future. Currently, due to other obligations elsewhere but also within Terkin, I don't think I will able to dedicate myself to that.

But in case I would touch that, should we call the directory `src/dist-packages` or `src/site-packages`, see [1]?

With kind regards,
Andreas.

[1] https://stackoverflow.com/questions/9387928/whats-the-difference-between-dist-packages-and-site-packages

**Comment by poesel** [comment]
I don't think it makes a difference here. So don't change anything. Only creates confusion.

Putting the mpys there, too, makes sense (although mpy are not strictly source files :) ). If everything is under one root directory you can sync it with pymakr.

No rush. I just wanted to know if we do it or not.
